tools/OresToFieldGuide/MainClass.cs

```diff
@@ -49,12 +57,21 @@ public static void Main(string[] args)
             ConsoleLogHelper.WriteLine(programArguments.ToString(), LogLevel.Message);
 
             var programInstance = new OresToFieldGuideProgram(programArguments);
-            var task = programInstance.Run();
-            while(!task.IsCompleted)
+            bool result = false;
+            try
             {
-            }
+                var task = programInstance.Run();
+                while(!task.IsCompleted)
```

Comment:
Task has a callback that'll run whenever it's finished, instead of a loop like this

Reply:
decided to just use the "Wait" method